packages/lesswrong/themes/globalStyles/globalStyles.ts

```diff
@@ -188,6 +188,29 @@ const commentsStyle = (theme: ThemeType): JssStyles => ({
   },
 });
 
+
+type Enumerate<N extends number, Acc extends number[] = []> =
+  Acc["length"] extends N
+    ? Acc[number]
+    : Enumerate<N, [...Acc, Acc["length"]]>;
+
+type IntRange<F extends number, T extends number> =
```

Comment:
Looks like this doesn't include the end number
<img width="646" alt="Screenshot 2023-11-24 at 11 15 22" src="https://github.com/ForumMagnum/ForumMagnum/assets/77623106/62c82039-1194-4eca-8c47-abeb9992161e">